src/index.tsx

```diff
@@ -10,6 +9,14 @@ const root = ReactDOM.createRoot(
 );
 root.render(
   <Provider store={store}>
+    <style>
+      @import
+      url('https://fonts.googleapis.com/css2?family=Carlito:ital,wght@0,400;0,700;1,400;1,700&display=swap')
+    </style>
```

Comment:
I think it will be better to move that imports to index.html file to head section:
```
    <link href='https://fonts.googleapis.com/css2?family=Carlito' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css2?family=Krona+One' rel='stylesheet' type='text/css'>
```